### ISSUE TITLE
README on dynamic method support presupposes very specific prior knowledge

### ISSUE THREAD

**Issue by mikebeaton** [FransBouma/Massive#291]
TLDR: I found the description of dynamic method support in README.markdown (the section **Named Argument Query Syntax**) very hard to understand. I've created a pull request to request updating the wording to make explicit some ideas that were very terse and implict before. I have tried very hard to keep my new wording short, sweet, and in the same style as the previous wording.

Longer:

I'm a C# programmer (for around 10 years, many other langauages before that), though I don't necessarily stay right on the bleeding edge of everything. I had never come across TryInvokeMember before Massive. And I've never used Rails. I don't think I'm an unreasonable demographic to use and work on Massive.

But I originally had *no* idea what "C#4's Method-on-the-fly syntax" meant, nor what an "ActiveRecord" is, And I was particularly confused by the whole reference to an earlier version ("Originally..."). Which version, where?

I hope these really aren't trivial changes, or changes for the lazy.

Yes, you can browse the Pulls, Issues and Commits history to work out what is going on. I did. If anyone else wants to look, search for 'Rails' in the Pull/Issue history, and see especially this commit https://github.com/FransBouma/Massive/commit/4ae06a0a2174d5c86667db0332498e15bdb993f5 . And  yes, you can use Google to search for ActiveRecord - but you still won't know what aspect of it is being referred to in this part of the readme, and why.

**Comment by FransBouma** [comment]
merged
